src/json-schema.ts

```diff
@@ -108,7 +108,7 @@ type PropertiesConstraint<
   : unknown;
 
 type AdditionalPropertiesConstraint<
-  AdditionalProperties extends SchemaLike | undefined
+  AdditionalProperties extends SchemaLike | (false | undefined)
```

Comment:
I didn't want to allow `additionalProperties: true`, so I'm specifically doing `(false | undefined)` instead of `(boolean | undefined)`

Reply:
We'll take this conversation to #30